db/models/User.js

```diff
@@ -0,0 +1,12 @@
+import mongoose from "mongoose";
+
+const { Schema } = mongoose;
+
+const userSchema = new Schema({
+  owner: { type: String, required: true },
+  favorites: { type: Array, required: true },
+});
+
+const User = mongoose.models.User || mongoose.model("User", userSchema);
+
+export default User;
```

Comment:
Don´t you also need the **connect .js** file here?

Reply:
It's already imported and connected

![Bildschirmfoto 2024-06-21 um 14 09 15](https://github.com/SabinePfanner/plants-app/assets/164749128/4fe1051f-1f19-4239-9fe2-9e252c00e203)